docs/content/deployment/run-launcher.mdx

```diff
@@ -3,54 +3,131 @@ title: Run Launchers | Dagster
 description: The run launcher is the interface to the computational resources that will be used to actually execute Dagster runs.
 ---
 
-# Run Launchers
+# Run launchers
+
+<Note>
+  This guide is applicable to Dagster Open Source (OSS) deployments. For Dagster
+  Cloud, refer to the <a href="/dagster-cloud">Dagster Cloud</a> documentation.
+</Note>
+
+Runs instigated from the Dagit UI, the scheduler, or the `dagster job launch` CLI command are launched in Dagster. This is a distinct operation from executing a pipeline using the `execute_pipeline` Python API or the CLI `execute` command. A launch operation allocates computational resources (e.g. a process, a container, a Kubernetes pod, etc) to carry out a run execution and then instigates the execution.
+
+The core abstraction in the launch process is the run launcher, which is configured as part of the [Dagster instance](/deployment/dagster-instance) The run launcher is the interface to the computational resources that will be used to actually execute Dagster runs. It receives the ID of a created run and a representation of the pipeline that is about to undergo execution.
+
+---
 
 ## Relevant APIs
 
 | Name                                                              | Description                   |
 | ----------------------------------------------------------------- | ----------------------------- |
 | <PyObject module="dagster._core.launcher" object="RunLauncher" /> | Base class for run launchers. |
 
-## Overview
-
-Runs instigated from the Dagit UI, the scheduler, or `dagster job launch` are "launched" in Dagster. This is a distinct operation from "executing" a pipeline using the `execute_pipeline` python API or the CLI `execute` command. A 'launch' operation allocates computational resources (e.g. a process, a container, a kubernetes pod, etc) to carry out a run execution and then instigates the execution.
-
-The core abstraction in the launch process is the run launcher, which is configured as part of the Dagster Instance. The run launcher is the interface to the computational resources that will be used to actually execute Dagster runs. It receives the ID of a created run and a representation of the pipeline that is about to undergo execution.
-
-## Default Run Launcher
+---
 
-The simplest run launcher is the built-in run launcher, <PyObject object="DefaultRunLauncher" />. This run launcher spawns a new process per run on the same node as the pipeline's repository location.
+## Built-in run launchers
 
-## Other Run Launchers
+The simplest run launcher is the built-in run launcher, <PyObject object="DefaultRunLauncher" />. This run launcher spawns a new process per run on the same node as the pipeline's code location.
```

Comment:
Pretty sure this was the only piece of copy that required an update for `Definitions`.

Reply:
pipeline --> job